src/batterystatus.h

```diff
@@ -84,7 +86,9 @@ class SYSTEMSETTINGS_EXPORT BatteryStatus : public QObject
     Q_ENUM(Status)
 
     ChargingMode chargingMode() const;
+    bool chargingOverride() const;
     void setChargingMode(ChargingMode mode);
+    void setChargingOverride(bool override);
```

Comment:
Interesting, I would expect it to reject use of the `override` keyword as an identifier.

Reply:
Did not even cross my mind... had to check. To me it looks like: "override" is not a "keyword." It is an "identifier with special meaning." Which apparently means it is treated as a regular identifier except in the very context where it is needed for tweaking overloading details or something. So, I guess, technically this would be ok.

Reply:
Anyway, as an side effect of renaming, the "override" argument is gone.